Managers/UInputMan.h

```diff
@@ -795,6 +795,28 @@ namespace RTE {
 		// Disallow the use of some implicit methods.
 		UInputMan(const UInputMan &reference) = delete;
 		UInputMan & operator=(const UInputMan &rhs) = delete;
+
+#ifdef __unix__
+		/// <summary>
+		/// Mouse input handler to circumvent the input drops that allegro does regularly, by replacing and disabling the default warping behaviour.
+		/// Motion events that are generated while the handler is working are offset such that the allegro driver doesn't mess up the mickeys.
+		/// This also handles the centering warp for relative mouse motion.
+		/// Should be applied to _xwin_input_handler. Might not run in the main thread, depending on how allegro was built.
+		/// </summary>
+		static void HandleAllegroMouseInput(void);
+
+		/// <summary>
+		/// Position the mouse on the screen in window coordinates. Generates MouseMotion events iff the requested position is different from the actual mouse position.
```

Comment:
Typo - `iff` should be `if`

Reply:
not technically a typo

Reply:
`if and only if`, I assume?
If so, I'd say expand it for clarity/obviousness. Even though I do actually know the term, I leaned towards assuming it was a typo lol